devolving_music/views/get_song_pair.py

```diff
@@ -29,14 +29,23 @@ def get(self, _request, event: Event):
         scores = SongScores(event)
         scores_list = scores.scores_list
 
+        info_list = SongScores.get_info_sort(scores_list)
+
+        # this is the threshhold at which we consider a song informed if we get a 
+        # informed song we find another informed song that is close in devolve space 
+        median_info = info_list[round(len(info_list)/2.0)].info_score
+
         # grab submission with a tendency to be low information
         score_low_info = SongScores.weighted_lowest_info(scores_list,LUCK_FACTOR)
         sub1 = score_low_info.song_submission
         # grab random submission if song is uninformed else grab the song closest in devolving space  
-        if(score_low_info.info_score >= INFORMED):
-            close_songs = scores.get_compare_submission_closest(score_low_info, INFORMED)
+        if(score_low_info.info_score >= median_info):
+            close_songs = scores.get_compare_submission_closest(score_low_info, median_info)
             closest_song = close_songs[0]
-            sub2= closest_song.song_submission
+            if(score_low_info.devolve_distance(closest_song) != inf):
+                sub2= closest_song.song_submission
+            else:
+                sub2 = scores.get_compare_submission_random(sub1.id)  
```

Comment:
Why do we need this if? It's hard to imagine the closest song to a well-informed song been infinitely far from it, but even if it is, isn't that as good as a random selection?

Reply:
i thought that too, but remember a song is infinetly far away if it has no information. With how sorting is set up when you initialize the play list with all no info songs. then every song will get the same song as the futherest song away in information space. Look at the screenshots for an example:
![image](https://user-images.githubusercontent.com/22030302/186924716-e29b6f30-d8a6-4a2d-b551-b7f878a92feb.png)
![image](https://user-images.githubusercontent.com/22030302/186924758-79e9fd0b-68cf-4f41-93be-c9f86bd8d705.png)
![image](https://user-images.githubusercontent.com/22030302/186924787-1c1fc006-c3aa-4b4f-bcb1-3bc4af67900f.png)
![image](https://user-images.githubusercontent.com/22030302/186924798-41bcb5e7-f3f5-400b-af31-850e55c20d31.png)
![image](https://user-images.githubusercontent.com/22030302/186924834-5dc281b9-4c05-4f0c-bd0e-f1d8b8c115ac.png)

Reply:
now with the if state ment 
![image](https://user-images.githubusercontent.com/22030302/186925183-e7cd85ed-6aba-49ca-8f06-fbab33c0161d.png)
![image](https://user-images.githubusercontent.com/22030302/186925284-88acba36-5124-4241-882a-8be9abfec9c7.png)
![image](https://user-images.githubusercontent.com/22030302/186925356-a19d7bc7-0611-472c-8003-1880035cc8ec.png)
![image](https://user-images.githubusercontent.com/22030302/186925420-d4b5c6e0-adc4-4e65-b3eb-67739a42b6c4.png)

Reply:
Now i could jumble all the infinetly far away songs in a similar way to how get information does, but i thought since this only happens at initilizization it's better just to have a simple if statement.

Reply:
or jumble score_list but I thought it's better this way